src/java.base/share/classes/sun/security/ssl/SSLSocketImpl.java

```diff
@@ -1686,7 +1691,7 @@ private void handleException(Exception cause) throws IOException {
 
         if (cause instanceof SocketException) {
             try {
-                conContext.fatal(alert, cause);
+                throw conContext.fatal(alert, cause);
```

Comment:
The fatal() will always throw an exception.  We could use the method without throw.  In some places, IDE or compiler could complain because it cannot detect that the fatal() method will throw and the program ends here.  In the past, we use fatal() method without throw, later we change the fatal() return value form "void" to an exception so that throw could be used to mitigate the IDE or compiler complaints.

It is not really necessary to make this code change here.  I just want to make the coding style consistent for the fatal() calling.

Reply:
Interesting, ok.  I don't recall seeing such situations with the IDE I often use.  (Netbeans)